packages/mds-utils/state-machine.ts

```diff
@@ -1,45 +1,58 @@
-import { VEHICLE_STATUSES, VEHICLE_EVENTS, VEHICLE_STATUS, VEHICLE_EVENT } from '@mds-core/mds-types'
+import { VEHICLE_STATUSES, VEHICLE_STATUS, VEHICLE_EVENT } from '@mds-core/mds-types'
 
+/* FIXME: Update to support Taxi */
```

Comment:
I'm surprised this isn't included in this PR.

Reply:
It is -- note to myself I forgot to remove :)